webapp/src/views/Collection.vue

```diff
@@ -126,7 +126,9 @@
   import Header from '@/components/Header.vue'
   import VideoThumbnail from '@/components/VideoThumbnail.vue'
   import Loading from '@/components/Loading.vue'
-
+  import store from '../store'
```

Comment:
Store is defined but never used. 
I think you should run lint on the webapp files if you have time.

Reply:
If you don't mind, we can lint after we merge to the development branch